x/mint/types/minter_test.go

```diff
@@ -63,25 +63,22 @@ func TestNextPhase(t *testing.T) {
 	blocksPerYear := uint64(100)
 	tests := []struct {
 		currentBlock, currentPhase, startPhaseBlock, blocksYear, expPhase uint64
+		currentSupply                                                     sdk.Int
+		targetSupply                                                      sdk.Int
 	}{
-		{1, 0, 0, blocksPerYear, 1},
-		{50, 1, 1, blocksPerYear, 1},
-		{99, 1, 1, blocksPerYear, 1},
-		{100, 1, 1, blocksPerYear, 1},
-		{101, 1, 1, blocksPerYear, 2},
-		// increase blocks_per_year
-		{102, 2, 101, blocksPerYear * 2, 2},
-		{201, 2, 101, blocksPerYear * 2, 2},
-		{299, 2, 101, blocksPerYear * 2, 2},
-		{300, 2, 101, blocksPerYear * 2, 2},
-		{301, 2, 101, blocksPerYear * 2, 3},
+		{1, 0, 0, blocksPerYear, 1, sdk.NewInt(10000), sdk.NewInt(14000)},
+		{50, 1, 1, blocksPerYear, 1, sdk.NewInt(12000), sdk.NewInt(14000)},
+		{99, 1, 1, blocksPerYear, 1, sdk.NewInt(13960), sdk.NewInt(11400000)},
```

Comment:
When we say `targetSupply` we mean target supply in this phase only, right?

Reply:
yes

Reply:
Okay I will add a test case that clarifies that.